layers/meta-opentrons/recipes-robot/opentrons-robot-app/files/opentrons-robot-app.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+if test -z "$XDG_RUNTIME_DIR"; then
+    export XDG_RUNTIME_DIR=/run/user/`id -u`
+    if ! test -d "$XDG_RUNTIME_DIR"; then
+        mkdir --parents $XDG_RUNTIME_DIR
+        chmod 0700 $XDG_RUNTIME_DIR
+    fi
+fi
+
+# wait for weston
+while [ ! -e  $XDG_RUNTIME_DIR/wayland-0 ] ; do sleep 0.1; done
+sleep 1
+
+/opt/opentrons-app/opentrons \
+    --disable-gpu \
+    --remote-allow-origins=* \
```

Comment:
this specifically is what was breaking devtools, this wasn't pluralized

Reply:
dev tools worked as expected.